src/main/java/schmidt/core/Schmidt.java

```diff
@@ -0,0 +1,307 @@
+            case "unmark":
+                try {
+                    markTaskHelper(tasks, trimmedInput, false);
+                } catch (NumberFormatException e) {
+                    print("The task number should be a number\n" +
+                            "\tunmark <task number>");
+                } catch (IndexOutOfBoundsException e) {
+                    print("Task number out of range\n" +
+                            "\tunmark <task number>");
+                } catch (SchmidtException e) {
+                    print(e.getMessage());
+                }
+                continue;
+            case "todo":
+                try {
+                    addTodoHelper(tasks, trimmedInput);
+                } catch (StringIndexOutOfBoundsException e) {
+                    print("The description of a todo cannot be empty\n" +
+                            "\ttodo <description>");
+                }
+                continue;
+            case "deadline":
+                try {
+                    addDeadlineHelper(tasks, trimmedInput);
+                } catch (SchmidtException e) {
+                    print(e.getMessage());
+                }
+                continue;
+            case "event":
+                try {
+                    addEventHelper(tasks, trimmedInput);
+                } catch (SchmidtException e) {
+                    print(e.getMessage());
+                }
+                continue;
+            default:
+                printValidCommands();
+            }
+        }
+    }
+
+    /**
+     * This is a function to add lines to print statements
+     */
+    public static void print(String message) {
+        System.out.println("------------------------------------------------------------");
+        System.out.println(message);
+        System.out.println("------------------------------------------------------------");
+    }
+
+    /**
+     * This is a function to print the valid commands
+     */
+    public static void printValidCommands() {
+        print("I'm sorry, but the valid commands are:\n" +
+                "\tbye\n" +
+                "\tlist\n" +
+                "\ttodo <description>\n" +
+                "\tdeadline <description> /by <date>\n" +
+                "\tevent <description> /from <date> /to <date>\n" +
+                "\tmark <task number>\n" +
+                "\tunmark <task number>");
+    }
+
+    /**
+     * This is a function to print the bye message
+     */
+    public static void printByeMessage() {
+        print("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * This is a helper method to print the list of tasks
+     *
+     * @param tasks The list of tasks
+     */
+    public static void printListHelper(ArrayList<Task> tasks) {
+        if (tasks.isEmpty()) {
+            print("You have no tasks in the list");
+            return;
+        }
+
+        System.out.println("------------------------------------------------------------\n" +
```

Comment:
Maybe you could consider declaring the "----..." as a constant class-level variable